user_guide/data_management_grids.rst

```diff
@@ -0,0 +1,306 @@
+
+- You can create a new grid view based on the existing one with *"Save as"*
+- You can rename an existing grid view with *"Rename"*
+- You can share the grid view with other users with *"Share"*. This means that other users will see your customized 
+  grid view in their grid view selector.
+- You can delete the grid view
+
+.. _user-guide-ui-components-grid-action-buttons:
+
+Other Grid Controls
+^^^^^^^^^^^^^^^^^^^
+
+You can adjust any grid view with action buttons - displayed right above the grid.
+
+|
+
+.. image:: ./img/data_management/grid/grid_action_buttons.png
+
+|
+
+- Change the amount of items displayed per page: :guilabel:`View Per Page`
+
+- Scroll pages (if there is more than one page of records): |ScrollPage|
+
+- Refresh the grid, i.e. get the newest details on the displayed records: |BRefresh|
+
+- Reset the grid, i.e. clear all the filters applied to the grid: |BReset|
+
+
+
+In some grids (as in the example), you can export all the details shown in the grid as a .csv file.
+
+
+.. _user-guide-ui-components-grid-action-icons:
+
+What You Can Do on a Grid
+-------------------------
+
+Create New Records from a Grid
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If permitted, you can initiate the creation of a new record directly from the grid.
+Right above the grid, there is usually a :guilabel:`Create New ...` button - click it, to get to the Create form.
+
+.. _user-guide-ui-components-grid-edit:
+
+Edit Records on a Grid
+^^^^^^^^^^^^^^^^^^^^^^
+
+If permitted, you can edit the records directly from the grid. By each cell there is an edit icon. Click it twice till 
+you edit the value. 
+
+|
+
+.. image:: ./img/data_management/grid/grid_edit.png
+
+|
+
+Click the check to confirm the change, or the cross to cancel it.
+
+|
+
+.. image:: ./img/data_management/grid/grid_edit_confirm.png
+
+
+.. _user-guide-ui-components-grids-delete-merge:
+
+Delete and Merge Records on a Grid
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If permitted, you can delete records.
+
+- To delete one record, hover the mouse over the action buttons (the last column) and click |IcDelete|.
+
+|
+
+.. image:: ./img/data_management/grid/grid_delete.png
+
+|
+
+
+- To delete several records (where available), check the record boxes (corresponding rows will be highlighted) and 
+  click the bulk delete action.
+
+|
+  
+.. image:: ./img/data_management/grid/grids_delete_bulk.png
+
+|
+
+In the same manner you can merge several records (if applicable). Currently, merge can only be done for several 
+:ref:`accounts <user-guide-accounts>`.
+
+
+Get to View Page and Edit Forms from a Grid
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If permitted, you can get to the :ref:`View Page <user-guide-ui-components-view-pages>` or 
+:ref:`Edit form <user-guide-ui-components-create-pages>` of any records in the grid.
+ 
+
+To get to the *"Edit"* page, hover the mouse over the action buttons (the last column) and click |IcEdit|.  
```

Comment:
Edit: To get to the *"Edit"* page, hover the cursor over the action buttons (the last column) and click |IcEdit|.